docs/notebooks/introduction.pct.py

```diff
@@ -270,16 +270,16 @@ def build_model(data):
 # %%
 bo = trieste.bayesian_optimizer.BayesianOptimizer(observer, search_space)
 
-result_mes: OptimizationResult = bo.optimize(15, initial_data, model, acq_rule)
+result_mes: OptimizationResult = bo.optimize(20, initial_data, model, acq_rule)
 
 if result_mes.error is not None: raise result_mes.error
 
 dataset_mes = result_mes.datasets[OBJECTIVE]
-mes_optimization_progress = np.minimum.accumulate(dataset_mes.observations)[5:]
-ei_optimization_progress = np.minimum.accumulate(dataset.observations)[5:]
+mes_optimization_progress = np.minimum.accumulate(dataset_mes.observations)
+ei_optimization_progress = np.minimum.accumulate(dataset.observations)
```

Comment:
does this mean we're also plotting the first 5? why the change?

Reply:
I decided it was helpful to see the starting point of each algorithm, i.e what happens before we switch to BO. It helps get an idea about useful BO is. What do you think?